ovh/resource_dbaas_logs_cluster.go

```diff
@@ -0,0 +1,213 @@
+package ovh
+
+import (
+	"fmt"
+	"log"
+	"net/url"
+	"strings"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func resourceDbaasLogsCluster() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceDbaasLogsClusterCreate,
+		Update: resourceDbaasLogsClusterUpdate,
+		Read:   resourceDbaasLogsClusterRead,
+		Delete: resourceDbaasLogsClusterDelete,
+		Importer: &schema.ResourceImporter{
+			State: resourceDbaasLogsClusterImportState,
+		},
+
+		Schema: resourceDbaasLogsClusterSchema(),
+	}
+}
+
+func resourceDbaasLogsClusterSchema() map[string]*schema.Schema {
+	schema := map[string]*schema.Schema{
+		"service_name": {
+			Type:     schema.TypeString,
+			Required: true,
+			ForceNew: true,
+		},
+		"cluster_type": {
+			Type:        schema.TypeString,
+			Description: "Cluster type",
+			Computed:    true,
+		},
+		"dedicated_input_pem": {
+			Type:        schema.TypeString,
+			Description: "PEM for dedicated inputs",
+			Computed:    true,
+		},
+		"archive_allowed_networks": {
```

Comment:
Put the non computed attributes before the "//Computed" line (for more visibility, readability and maintability)